package.json

```diff
@@ -27,9 +27,10 @@
     "sveltejs"
   ],
   "scripts": {
-    "build": "pnpm run build:meta && pnpm run build:ts",
+    "build": "pnpm run build:meta && pnpm run build:ts && pnpm run build:types",
     "build:meta": "pnpm run ts ./tools/update-meta.ts",
     "build:ts": "tsc --project ./tsconfig.build.json",
+	"build:types": "pnpm run ts ./tools/update-rule-types.ts && copyfiles -f ./src/rule-types.d.ts ./lib",
```

Comment:
Do we need a copy of the file? Doesn't the copied file get created just by running `tsc`?

Reply:
Ah, since `tsdoc` is removed, we are copying the file.
Let's remove `removeComments` from `tsconfig.build.json`.